openupgrade_scripts/scripts/mrp/18.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2025 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+add_columns = [
+    ("product.document", "attached_on_mrp", "char", "hidden"),
+]
+
+copy_columns = {
+    "mrp_production": [
+        ("location_dest_id", "location_final_id", "integer"),
```

Comment:
are you sure we don't need to do some adjustment to `location_dest_id` or `location_final_id`? :)

Reply:
well not entirely, I find the way this in implemented in v18 somewhat confusing. My understanding is that `location_final_id` is mainly used for reporting and UI purposes, while `location_dest_id` is the relevant one for stock. Will add a few tests comparing migrated v17 and v18 behavior